repository-data/application/src/main/resources/hcm-config/configuration/modules/documentworkflow.scxml

```diff
@@ -0,0 +1,596 @@
+  -->
+<scxml version="1.0"
+       xmlns="http://www.w3.org/2005/07/scxml"
+       xmlns:hippo="http://www.onehippo.org/cms7/repository/scxml"
+       xmlns:cs="http://commons.apache.org/scxml">
+
+    <script>
+        def getScxmlId() { workflowContext.scxmlId }
+        // draft variant property method
+        def getDraft() { workflowData.documents['draft'] }
+        // unpublished variant property method
+        def getUnpublished() { workflowData.documents['unpublished'] }
+        // published variant property method
+        def getPublished() { workflowData.documents['published'] }
+        // current user property method
+        def getUser() { workflowContext.user }
+        // current requests map property method
+        def getRequests() { workflowData.requests }
+        // returns the variant for copying and checking copy access privileges
+        def getCopySource() { published ?: unpublished ?: null }
+        // returns the variant for deleting and checking delete privileges
+        def getDeleteSource() { unpublished ?: published ?: draft }
+        // returns the current draft holder (editor) if defined (and draft exists)
+        def getHolder() { draft?.holder }
+        // true if draft exists and currently being edited
+        def boolean isEditing() { !!holder }
+        // true if draft exists and edited by current user
+        def boolean isEditor() { holder == user }
+        // true if draft exists and not currently editor or edited by current user
+        def boolean isEditable() { !holder or editor }
+        // true if published variant exists with availability 'live'
+        def boolean isLive() { published?.isAvailable('live') }
+        // true if unpublished variant exists with availability 'preview'
+        def boolean isPreview() { unpublished?.isAvailable('preview') }
+        // true if either unpublished or published variant exits with availability 'preview'
+        def boolean isPreviewAvailable() { unpublished?.isAvailable('preview') or published?.isAvailable('preview') }
+        // true if unpublished variant exists and no published variant exists with availability 'live'
+        //                                        or they have a different lastModified
+        def boolean isModified() { unpublished and (!live or unpublished.lastModified!=published.lastModified) }
+        // true if there is an outstanding workflow request
+        def boolean isRequestPending() { workflowData.requestPending }
+        def getContainingFolder() { workflowData.handle.parent }
+    </script>
+
+    <!-- the initial no-document state is used to prevent entering the handle state if there is no document -->
+    <state id="no-document">
+        <!-- automatic transition to and continue in parallel handle state only when a document variant exists -->
+        <transition target="handle" cond="!workflowData.documents.isEmpty()"/>
+    </state>
+
+    <parallel id="handle">
+
+        <!--  -->
```

Comment:
In the original this comment says `<!-- the atomic status state is used to report several statuses and info feedback to the invoking workflow -->` - has it been removed intentionally?

Reply:
Good spot, no idea how I did that, have reinstated